src/main/java/com/google/research/bleth/simulator/Board.java

```diff
@@ -1,15 +1,17 @@
 package com.google.research.bleth.simulator;
 
-import static com.google.common.base.Preconditions.checkArgument;
-import static com.google.common.base.Preconditions.checkNotNull;
-
 import com.google.common.collect.ArrayTable;
 import com.google.common.collect.ImmutableList;
+import com.google.gson.Gson;
+
```

Comment:
Unnecessary blank line.
https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing